### DISCUSSION TITLE
Are you absolutely sure you know how to use the button element?

### DISCUSSION THREAD

**Discussion by stevepolitodesign** [therubyradar/submissions#181] [Submissions]
I used to think the `<button>` element could only exist inside a `<form>` and that it did not support any attributes. In other words, I basically thought it was a more semantic version of `<input type="submit">`.

However, after working on a payment form for a client project, I realized that the `<button>` element is a robust, versatile and under utilized tool. Plus, knowing how to leverage forms and buttons will benefit you when using [Turbo](https://thoughtbot.com/blog/dynamic-forms-with-turbo) and [Stimulus](https://thoughtbot.com/blog/dynamic-forms-with-stimulus).

https://thoughtbot.com/blog/button-element-tutorial

**Comment by cjilbert504** [comment]
Thanks @stevepolitodesign!!!

**Reply by stevepolitodesign** [reply]
Thank you for including it!
